src/artrace.py

```diff
@@ -83,7 +81,9 @@ def dispdir(msframe, dispwin=None, mode=0):
             return 1
 
 
-def trace_object(slf, det, sciframe, varframe, crmask, trim=2.0, triml=None, trimr=None, sigmin=2.0, bgreg=None, maskval=-999999.9, order=0):
+def trace_object(slf, det, sciframe, varframe, crmask, trim=2.0,
+                 triml=None, trimr=None, sigmin=2.0, bgreg=None,
+                 maskval=-999999.9, order=0, doqa=True, debug=False):
```

Comment:
New to me! :)
I've expanded on this and modified it.  
Let me know if you are ok with it.